frontend/src/InteractiveSearch/InteractiveSearchRow.css

```diff
@@ -72,6 +72,9 @@
   height: 20.39px;
   vertical-align: middle;
   line-height: 20.39px;
+  &:hover {
```

Comment:
The build failed with: `Expected empty line before rule  rule-empty-line-before`, just needs a new line before it.

Reply:
done, I was sure I ran `stylelint`!
any reason we don't have a `stylelint-fix` command in `package.json`?